src/main/java/nextstep/ladder/main.java

```diff
@@ -0,0 +1,12 @@
+package nextstep.ladder;
+
+import nextstep.view.InputView;
+import nextstep.view.ResultView;
+
+public class main {
```

Comment:
코딩 컨벤션 확인 부탁 드립니다 😄 
```suggestion
public class Main {
```
그리고 단순히 `Main`보다 좀 더 의미있는 이름이면 더 좋을 것 같습니다 
`LadderGameMain`이라던지 `LadderGameRunner`라던지 😅 
늘 이름 짓는 건 고민인 것 같습니다 😭 🙇

Reply:
메인도 이제 좀의미부여해서 짓도록하겠습니다.. 이름짓는거 항상고통스러워요...